src/server/bg_services/agent_blocks_verifier.js

```diff
@@ -70,28 +70,19 @@ class AgentBlocksVerifier {
             });
     }
 
-    populate_agent_blocks_verifier_blocks(blocks) {
+    /**
+     * 
+     * @param {nb.BlockSchemaDB[]} blocks 
+     */
+    async populate_agent_blocks_verifier_blocks(blocks) {
         if (!blocks || !blocks.length) return;
 
-        return this.populate_nodes_for_blocks(blocks)
-            .then(blocks_with_nodes => {
-                const blocks_with_alive_nodes = blocks_with_nodes.filter(block =>
-                    block.node && block.node.rpc_address && block.node.online);
-                if (!blocks_with_alive_nodes || !blocks_with_alive_nodes.length) return;
-                return this.populate_chunks(blocks_with_alive_nodes, 'chunk', { frags: 1, chunk_config: 1 });
-            })
-            .then(populated_blocks => {
-                if (!populated_blocks || !populated_blocks.length) return;
-                return populated_blocks.map(block => {
-                    const chunk_config = this.get_by_id(block.chunk.chunk_config);
```

Comment:
Notice that we've checked if chunk_config exists and did not throw in case that it doesn't. 
This is something that you must verify that works with tests and regular case. 
Since as far as I know you currently throw if it doesn't exist in chunk_populate.
Please check it.

Reply:
no need anymore. used ccc for block_md digest_type. now we mock just it.